base.Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Dockerfile to bootstrap build and test in openshift-ci
 
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
```

Comment:
would it make sense to split this out of the PR? Or is it a pre-requisite for the new behaviour?

Reply:
Sounds like a plan, reverted, it shouldn't be required by any of the changes in this PR.